src/Components/SightingsFilter/SightingsFilter.js

```diff
@@ -0,0 +1,149 @@
+            <h1>Name: {`${sighting.name}`}</h1>
+            <h1>City: {`${sighting.city}`}</h1>
+            <h1>Event Type: {`${sighting.event_type}`}</h1>
+            <h1>Description: {`${sighting.description}`}</h1>
+            <h1>Comments:</h1>
+            {/* populate when BE has comments` */}
+            <div className="comments-div">
+              <h1 className="single-comment">
+                USERNAME: holy crap this ia a lot of comments, so many comments
+                about stuff
+              </h1>
+              <h1 className="single-comment">
+                USERNAME: holy crap this ia a lot of comments, so many comments
+                about stuff
+              </h1>
+              <h1 className="single-comment">
+                USERNAME: holy crap this ia a lot of comments, so many comments
+                about stuff
+              </h1>
+              <h1 className="single-comment">
+                USERNAME: holy crap this ia a lot of comments, so many comments
+                about stuff
+              </h1>
+              <h1 className="single-comment">
+                USERNAME: holy crap this ia a lot of comments, so many comments
+                about stuff
+              </h1>
+              <h1 className="single-comment">
+                USERNAME: holy crap this ia a lot of comments, so many comments
+                about stuff
+              </h1>
+              <h1 className="single-comment">
+                USERNAME: holy crap this ia a lot of comments, so many comments
+                about stuff
+              </h1>
+              <h1 className="single-comment">
+                USERNAME: holy crap this ia a lot of comments, so many comments
+                about stuff
+              </h1>
+              <h1 className="single-comment">
+                USERNAME: holy crap this ia a lot of comments, so many comments
+                about stuff
+              </h1>
+              <h1 className="single-comment">
+                USERNAME: holy crap this ia a lot of comments, so many comments
+                about stuff
+              </h1>
+            </div>
+          </div>
+        </article>
+      );
+    });
+  };
+
+  const handleFilterChange = (event) => {
+    setSelectedState(event.target.value);
+  };
+
+  const displayFilteredSightings = () => {
+    const filteredSightings = filterSightings(selectedState);
+    return generateFilteredSightings(filteredSightings);
+  };
+
+  return (
+    <main className="filter-main">
+      <h1 className="glitch" data-text="UFOMG">
+        UFOMG
+      </h1>
+      <div className="filter-div">
+        <aside className="filter-aside">
+          <h3>Show Sightings By State</h3>
+          <select
+            value={selectedState}
+            onChange={handleFilterChange}
+            className="drop-down"
+          >
+            <option disabled={selectedState ? true : false}>
+              -- Choose State --
```

Comment:
not sure what this  '-- choose state---' is, If it is a comment we will need to delete